pegasus/sites.v3/hourofcode.com/i18n/public/bn/tos.md

```diff
@@ -1,6 +1,6 @@
 ---
 
-শিরোনামঃ সেবার শর্ত সমূহ
+title: সেবার শর্ত সমূহ
```

Comment:
I'm curious to try two things

1) Make this change in i18n/locales/bn-BD/hourofcode/tos.md. Since this file isn't being updated by crowdin anymore, I don't think the sync would overwrite my change. And once I fix the file, then it would stop updating the file in pegasus incorrectly.

2) Deleting all of the tos files in i18n/locales and pegasus/sites.v3/hourofcode.com/i18n/public.

What do you think? I'm happy to make the changes, if you agree.

Reply:
those both sound like good ideas to me! Thanks for taking this on